src/main/fc/config.c

```diff
@@ -631,6 +631,10 @@ void validateAndFixConfig(void)
     validateAndFixGyroConfig();
 #endif
 
+#ifdef RUNCAM_SPLIT_SUPPORT
```

Comment:
Yes, you are right, the macro name USE_RCSPLIT is better then RUNCAM_SPLIT_SUPPOR, I'll change it in next commit.